trade-finance-manager-ui/server/helpers/decode-html-entities.ts

```diff
@@ -0,0 +1,15 @@
+import * as cheerio from 'cheerio';
+
+/**
+ * Decodes HTML entities in the provided string.
+ * @param input - The string containing HTML entities to decode.
+ * @returns The decoded string with HTML entities converted to their
+ * corresponding characters, or undefined if input undefined.
+ */
+export const decodeHtmlEntities = (input?: string): string | undefined => {
+  if (!input) {
+    return input;
+  }
+
+  return cheerio.load(input).text();
```

Comment:
1. Nice work, can we standardise this per EXIP https://github.com/UK-Export-Finance/exip/blob/main/src/api/helpers/replace-character-codes-with-characters/index.ts please thought might need to expand on the special characters please.

2. Can we add this to `libs/common` as it will be super useful for other microserivces.

Reply:
1. I agree that generally we should try to standardise things, but I think the right approach here, and best practice, is to make use of a library to do this rather than trying to handroll something ourselves.
\
The implementation in this PR uses cheerio which I used as it is already in the TFM UI packages and has ~9m weekly downloads, is well maintained, and should be well tested.
\
Should we not be using a package like this that already handles all the possible HTML entities (rather than a subset) and already has its own tests for these?
\
I think the linked EXIP function should also adopt this approach of using a package longer term

2. Yep good point, will make that change thanks
\
Cheerio isn't a dependency in libs/common at present and the [html-entities package](https://www.npmjs.com/package/html-entities) (~18m weekly downloads) is more suitable for this decoding so I'd prefer to use that (but this is dependent on the above).

Reply:
Thanks @sw-mattw lets go with point 2 please using `html-entities-package`. Thanks

Reply:
Just found this in `utils/data-migration/bss-ewcs/migrate-deals.js` thanks - I'll update that to use the libs/common function and remove the package from utils